addons/mail/static/src/models/follower/follower.js

```diff
@@ -157,6 +157,11 @@ function factory(dependencies) {
                     method: 'message_subscribe',
                     args: [[this.followedThread.id]],
                     kwargs,
+                }).then(() => {
```

Comment:
Avoid using `then` explicitly when possible, here you could make the anonymous function `async () => {`
and use `await` on the rpc call.

Reply:
https://github.com/odoo/odoo/blob/18b790c966719c6756090d33b35f424b70b73cbc/addons/mail/static/src/models/follower/follower.js#L155

This is already an async and await function, and now we are also not displaying a notification on the removal of the user.
and now we are displaying the notification after already awaited rpc call.

Reply:
Ok thank you. I posted on the task because I think the counter change is too subtle when removing someone else. We probably want a notification in that case too, let's see answer from PO. But otherwise current diff is fine if we want to keep this behavior.